.cirrus.yml

```
@@ -0,0 +1,49 @@
+task:
+  trigger_type: manual
+  name: E2E/Integration Tests
+  timeout_in: 45
+  container:
+    image: ubuntu:21.04
+    kvm: true # required for E2E/Integrations tests
+    cpu: 4
+    memory: 8G
+  env:
+    DEBIAN_FRONTEND: noninteractive
+    CI: true
+    FORCE_COLOR: 1 # force terminal color
+    CIRRUS_CLONE_DEPTH: 0 # Required to get app version
+    PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1 # disable auto-browser download
+    KEYRING: /usr/share/keyrings/nodesource.gpg
+    NODE_VERSION: node_14.x # set node version e.g: "node_16.x" for Node 16 LTS
+    DISTRO: hirsute # release codename for Ubuntu 21.04 - see "lsb_release -s -c"
+  install_deps_script: |
+    apt-get update
+    apt-get install -y --no-install-recommends ca-certificates gcc g++ curl git golang openssh-client make netcat sudo vim gpg \
+                    libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
+    curl -sLf -o /dev/null "https://deb.nodesource.com/"$NODE_VERSION"/dists/"$DISTRO"/Release"
+    curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | gpg --dearmor | tee "$KEYRING" >/dev/null
```

Why are we doing `… | tee foo >/dev/null`? (This would have made sense if we required `sudo`, but we're already root…)
```suggestion
    curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | gpg --dearmor > "$KEYRING"
```

---

Moving back fourth from root <> non-root... my brain frozen LOL
Thanks for catching.